apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -144,6 +150,15 @@ const BookingPage = ({
         collectPageParameters("/book", { isTeamBooking: document.URL.includes("team/") })
       );
     }
+    reserveSlot();
+    const interval = setInterval(
+      reserveSlot,
+      parseInt(process.env.NEXT_PUBLIC_MINUTES_TO_BOOK || "5") * 60 * 1000 - 2000
+    );
+    return () => {
+      clearInterval(interval);
+      releaseSlotMutation.mutate();
+    };
```

Comment:
this is to keep the slot accupied when the user stays in the booking form and release it when he leaves.